rules_conditions.md

```diff
@@ -0,0 +1,226 @@
+# Refinery Conditions
+
+## Overview
+
+Refinery rules are described as a series of conditions.
+Each condition is composed from a combination of these named parameters:
+
+- `Field` (or `Fields`)
+- `Operator`
+- `Value`
+- `Datatype`
+
+The `Operator` is never optional, and controls which of the parameters are required and which are optional.
+
+## `Field`
+
+The `Field` parameter points to a specific named element in the trace data.
+If a `Field` is named within a span, then it `exists`.
+A specific `Field` may or may not exist on any specific span in a trace.
+It might not even exist within a trace at all.
+
+When a `Field` is absent in all spans within a trace, the associated rule does not apply to that trace.
+
+A `Field` is always a single string
+A `Field` is always matched by exact comparison.
+No transformations for case or punctuation are performed.
+
+### Example use of `Field`
+
+```yaml
+Conditions:
+    Field: http.route
+    Operator: =
+    Value: /health-check
+```
+### Leveraging Special Refinery Telemetry in Root Spans
+
+Some Refinery configuration options introduce special names that are added to telemetry.
+
+For example, when `AddCountsToRoot` is enabled, `meta.span_count` is added to all root spans, allowing for the creation of rule conditions based on span counts.
+
+```yaml
+Conditions:
+    Field: "meta.span_count"
+    Operator: ">"
+    Value: 300
+    Datatype: int
+```
+
+In this scenario, the rule applies to traces with more than 300 spans.
+For details about all supported special fields, check out [documentation here](https://docs.honeycomb.io/manage-data-volume/refinery/configuration/#refinery-telemetry)
+
+### Virtual Fields
+
+To handle specific scenarios when rules are evaluated before the arrival of root spans, Refinery introduces the concept of virtual fields. These fields provide metadata about traces that have timed out while waiting for their root span.
+
+```yaml
+Rules:
+    - Name: Drop any big traces
+      Drop: true
+      Conditions:
+        Field: "?.NUM_DESCENDANTS"
+        Operator: ">"
+        Value: 1000
+        Datatype: int
+
+```
+
+This example shows a rule that drops traces containing more than 1000 spans, using the virtual field `?.NUM_DESCENDANTS`.
+
+#### Supported Virtual Fields
+
+All virtual fields will be prefixed with `?.` to distinguish them from normal fields.
```

Comment:
We're not living in the future. We're talking about what Virtual Fields does right now.

```suggestion
All virtual fields are prefixed with `?.` to distinguish them from normal fields.
```